projects/admin/src/manual_translations.ts

```diff
@@ -132,3 +132,15 @@ _('public_note');
 _('staff_note');
 _('checkin_note');
 _('checkout_note');
+
+// Local fields
+_('local_field_1');
```

Comment:
Perhaps they are already in the JSONSchema, no?

Reply:
I don't have these keys in the JSON Schema, because these names are properties.